sdks/python/README.md

```diff
@@ -448,11 +339,7 @@ p.run()
 
 ### Combiner Examples
 
-Combiners are used to create a `PCollection` that contains the sums
-(or max or min) of each of the keys in the initial `PCollecion`.
-Such standard Python functions can be used directly as combiner
-functions. In fact, any function "reducing" an iterable to a
-single value can be used.
+Combiner transforms use functions such as sum, min, or max (or any "reducing" function) to combine multiple values of a PCollection into a single value.
```

Comment:
The description for "reducing" function is lost here. (function "reducing" an iterable to a single value can be used).